Makefile

```diff
@@ -55,13 +55,13 @@ push:
 
 .PHONY: ovnk-deploy
 ovnk-deploy:
-	sed -e 's~quay\.io/netobserv/goflow-kube:dev~$(IMAGE):$(VERSION)~' ./examples/goflow-kube.yaml | kubectl apply -f - && \
+	sed -e 's~quay\.io/netobserv/goflow2-kube:main~$(IMAGE):$(VERSION)~' ./examples/goflow-kube.yaml | kubectl apply -f - && \
```

Comment:
Should the prow release process be also responsible for creating this image tags?
Or should we have different release process?

Reply:
Not sure to understand, this target is intended for local dev/test purpose where you can use your own image/tag, so I would leave prow out of that.

Or do you suggest that prow updates the sample YAML to make it point to the latest release?

IMO we can keep prow just for the release process and keep also the `main` tag that is more useful for dev/testing and is still created from a quay trigger